app/Exceptions/Handler.php

```diff
@@ -63,6 +63,12 @@ public function render($request, Throwable $exception)
         return parent::render($request, $exception);
     }
 
+    public function report(Throwable $e)
+    {
+        parent::report($e);
+        \Flare::report($e);
```

Comment:
Is this really needed?

Reply:
I think so, easy to test.  If I remember, when calling report() it was not sending it.

Reply:
Not needed, removed :)